src/document.jl

```diff
@@ -119,10 +122,14 @@ function get_position_at(doc::Document, offset::Integer)
     offset > sizeof(doc._content) && error("offset[$offset] > sizeof(content)[$(sizeof(doc._content))]")
     line_offsets = get_line_offsets(doc)
     line, ind = get_line_of(doc._line_offsets, offset)
+    ind = nextind(get_text(doc), ind)
     char = 0
-    while offset > ind
-        ind = nextind(doc._content, ind)
+    while offset >= ind
+        if nextind(get_text(doc), ind) - ind > 2
```

Comment:
Same story here, I don't think having more than 2 codepoints in UTF-8 is the right cut-off, if I understand UTF-8 and UTF-16 correctly.